lib/vehicles/american.vehicle.ts

```diff
@@ -208,7 +209,7 @@ export default class AmericanVehicle extends Vehicle {
         charging: vehicleStatus?.evStatus?.batteryCharge,
         batteryCharge: vehicleStatus?.battery?.batSoc,
       },
-    };
+    } as VehicleStatus;
```

Comment:
I think this is redundant somewhat as the return type of the method is a `VehicleStatus`. Nevertheless, it doesn't hurt anything keeping it in.

Reply:
While developing its needed to hightlight any missing properties etc